jmix-quartz/quartz-flowui/src/main/java/io/jmix/quartzflowui/event/QuartzJobEndEvent.java

```diff
@@ -16,11 +16,11 @@
 
 package io.jmix.quartzflowui.event;
 
-import com.vaadin.flow.component.Component;
+import org.quartz.JobDetail;
 import org.springframework.context.ApplicationEvent;
 
-public class QuartzDataChangedEvent extends ApplicationEvent {
-    public QuartzDataChangedEvent(Component source) {
+public class QuartzJobEndEvent extends ApplicationEvent {
+    public QuartzJobEndEvent(JobDetail source) {
```

Comment:
Store execution context and exception

Reply:
added

Reply:
Where?
Add field for JobExecutionContext with explicit getter and use it in event handler.
JobDetail field is not required.

Reply:
ok